kong/runloop/handler.lua

```diff
@@ -773,13 +774,14 @@ return {
       if ctx.KONG_PROXIED == true then
         -- Report HTTP status for health checks
         if balancer_data and balancer_data.balancer and balancer_data.ip then
```

Comment:
shouldn't this test now test for `balancer_handle` instead of `ip`?

Reply:
Either one is ok